paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -76,13 +87,13 @@ class FormViewModel(
 
     private val paramKey: MutableMap<String, Any?> = formDataObject.paramKey
     private val types: List<Field> = formDataObject.allTypes
-    private val elementMap = mutableMapOf<Field, TextFieldElement>()
+    private val elementMap = mutableMapOf<Field, Element>()
```

Comment:
You might be able to simplify this with:

```
    private val elementMap = types.associateWith { type ->
        when (type) {
            Field.NameInput -> TextFieldElement(NameConfig()) // All configs should have the label passed in for consistency
            Field.EmailInput -> TextFieldElement(EmailConfig())
            Field.CountryInput -> DropdownElement(CountryConfig())
        }
    }
```

The nice thing about `associateWith` is it also preserves the iteration order, so you can use `elementMap.values` to populate `listCompleteFlows` and `listOfPairs`.

Reply:
done